pylint/test/functional/undefined_variable.py

```diff
@@ -232,3 +232,16 @@ class LambdaClass5:
     myattr = 1
     mylambda = lambda: LambdaClass5.myattr
     mylambda4 = lambda a=LambdaClass5: lambda: a # [undefined-variable]
+
+
+def Main():
```

Comment:
Let's rename to function to be clearer where this is coming from (something along the lines of `undefined_variable_issue_...` which should make it more obvious what it is testing.)